os/arch/arm/src/amebasmart/amebasmart_idle.c

```diff
@@ -79,10 +80,20 @@ static void up_idlepm(void)
 		/* MCU-specific power management logic */
 		switch (newstate) {
 			case PM_NORMAL:
+#ifdef CONFIG_PM_DVFS
+				up_set_dvfs(0);
+#endif
 				break;
 			case PM_IDLE:
+#ifdef CONFIG_PM_DVFS
+				up_set_dvfs(1);
```

Comment:
What kind of Sinairo is this?

We also use the highest freq of the highest cpu for IDLE and STANDBY.

Does this return to the top freq on the next tick irq?

Reply:
This is added for saving power in different state. I mean we do it in idle thread because PM state change is expected when application has lock/unlock to the idle domain.
Since IDLE and STANDBY still use highest frequency, then which scenario apply DVFS?

Reply:
Thank you for suggesting
There is no DVFS scenario that we are reviewing yet.